### PR TITLE
Xss post

### PR DESCRIPTION
**Pull request by HelenaEksler**

### CODE REVIEW

**Comment by HelenaEksler** [comment]
cross site scripting attack detected
selection_067

**Comment by HelenaEksler** [comment]
@amitaibu Could you check? Maybe need to add/change/remove something.

File: _config.yml [hunk 1]
```diff
@@ -78,10 +78,10 @@ authors:
       name: Helena Eksler
       title: Team Lead
       is_member: true
-      bio: Something about Helenas
+      bio: Cooking well, doesn't matter - dinner or code.
       email: helena@gizra.com
       github: HelenaEksler
-      avatar: female.jpg
```
Comment (amitaibu) [inline _config.yml]: oh my :(

Comment (HelenaEksler) [inline _config.yml]: yeah ) I've added avatar

**Comment by amitaibu** [comment]
* Edited text. @HelenaEksler please go over it, and let me know if you think it's ok
* I've reduced images to 800px width
* Can you please change the thumbnail image, it's a little ugly ;) (and also needs to be smaller)

Other than that - nice work :)

**Comment by HelenaEksler** [comment]
selection_069

This picture is better, I think.

**Comment by amitaibu** [comment]
> This picture is better, I think.

Ok, but please reduce the size of it to be like the thumbnail

**Comment by amitaibu** [comment]
@niryariv ready for review

File: app/_posts/2016-02-24-xss-attack-caught.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+One of the major security concerns in websites and web applications is Cross Site Scripting (XSS). You definitely don't want somebody to run their own malicious code in your website.
+
+We all know about Drupal's `check_plain()`, `filter_xss()` and similar functions that sanitize user generated text, but unless we are actively checking for XSS, how can we be sure we've added them on all the right places?
+
+Well, we found a nice solution for it that can be easily applied in your projects as well.
+
+<!-- more -->
+
+All Gizra's projects are scaffolded from our home grown [hedley generator](/content/yo-hedley/), which comes with migration files that help with adding dummy and real content to your Drupal site. What we do when we add our own dummy migration data, so every time we rebuild the site it comes with some content, is add some data that has deliberate XSS in the title, body and text fields.
```
Comment (niryariv) [inline app/_posts/2016-02-24-xss-attack-caught.md]: @HelenaEksler keep in mind the reader isn't aware of our migration files methodology, perhaps explain a bit.

Basically, if I understand, you already populate the site with dummy content for testing and getting an idea of what it looks like. The new thing is that you pre-inject that dummy content with some XSS attacks so you can spot vulnerabilities. That's neat stuff - kind of like a vaccine perhaps, or testing water leaks etc. If you find a good metaphor from the real world you can make it much more accessible to the reader. Also show what it actually looks like (I assume ```alert("XSS");``` or something?)

**Comment by amitaibu** [comment]
@HelenaEksler ready? :)

**Comment by HelenaEksler** [comment]
@amitaibu @niryariv As to me - it's ready. But could you check one more time :)

**Comment by amitaibu** [comment]
No worries, I always give it a final look. Can't resist it ;)

**Comment by amitaibu** [comment]
I've edited a bit more the text, reduced the images size and moved the first image to the teaser area.

Thanks!

**Comment by HelenaEksler** [comment]
Thanks!

**Comment by amitaibu** [comment]
Deployed -- http://www.gizra.com/content/xss-attack/

**Pull request merged**